de.ient.YUView.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: de.ient.YUView
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: YUView
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/bin/x86_64-linux
+  - --env=QT_NO_FT_CACHE=1
+build-options:
+  env:
+    LD_LIBRARY_PATH: /app/lib:/app/lib64
+
+modules:
+  - name: YUView
+    buildsystem: qmake
+    sources:
+      - type: git
+        branch: deployment
+        url: https://github.com/IENT/YUView.git
+        commit: 64ccb50b85eed1db9d44b7f26c960b0807927f33
+
+  - name: libde265
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=release
+      - -DONLY_LIB=ON
+      - -DCMAKE_INSTALL_LIBDIR:PATH=/app/lib
```

Comment:
This should not be needed.

Reply:
I can remove 
`
      - -DCMAKE_BUILD_TYPE=release
`

`
       - -DONLY_LIB=ON 
`
is required in order to only build the library and not the example programs.

If I remove
`
       - -DCMAKE_INSTALL_LIBDIR:PATH=/app/lib 
`
I get
`
CMake Error at libde265/CMakeLists.txt:89 (INSTALL):
  install TARGETS given no LIBRARY DESTINATION for shared library target
  "de265".
`
The related part in the Cmake is
```
   INSTALL(TARGETS
     #libaries
     ${LIBDE265_LIBRARY_NAME}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   )
```
Should  CMAKE_INSTALL_LIBDIR be set automatically by flatpak?